addons/mass_mailing/tests/common.py

```diff
@@ -233,6 +233,16 @@ def setUpClass(cls):
 
         cls.email_reply_to = 'MyCompany SomehowAlias <test.alias@test.mycompany.com>'
 
+    @classmethod
+    def _create_partner(cls, partner_count=5):
+        return cls.env['res.partner'].create([{
+            'name': 'AutoPartner_%04d' % (x),
+            'email': tools.formataddr((
+                'AutoPartner_%04d' % (x),
+                'partner_email_%04d@example.com' % (x),
```

Comment:
Check also other occurrence in your code
```suggestion
                'partner_email_%04d@example.com' % x,
```